docroot/themes/custom/mass_theme/overrides/css/basic-html.css

```diff
@@ -115,3 +115,12 @@ form.fsForm.rtl {
 .page-content--intro .ma__video__skip-link {
   background-color: #fff;
 }
+
+.ma__rich-text h4 {
+  clear: both;
+}
+
+.ma__rich-text div.align-left.half-width.embedded-entity + ul {
+  margin-left: 83px;
```

Comment:
This is too specific, the image could be at any size, float right or left, also we need to consider different screen sizes
For instance, after applying this margin, at a smaller screen size, the image takes up the full row however the bulleted list still has a margin-left of 83px
![Screenshot 2024-08-22 at 11 28 00 AM](https://github.com/user-attachments/assets/4b19f5e2-3047-4dd8-b7b3-92e5f141c8f1)

TBH, I think there are two acceptable fixes:
1. Clear float for a bulleted list (same as the headings). I don't think how often we see a bulleted list side by side with an image like this - The example in openmass is an edge case, also a misuse of bulleted list. We should tell them not to use bulleted list this way
2. If we really anticipate there's a need to have a bulleted list side by side with a floating image, we could add `list-style-position: inside` instead of adding a margin-left at a specific value. This moves the bullet inside the content. This is not a perfect fix but will avoid the overlap. I think this fix should also exist in Mayflower and take into consideration the media queries. 

Mayflower test: http://localhost:3000/?p=organisms-rich-text
Before:
![Screenshot 2024-08-22 at 5 57 58 PM](https://github.com/user-attachments/assets/4898f783-4f8d-4ed4-801f-8810ff6c4e95)
After:
![Screenshot 2024-08-22 at 5 58 03 PM](https://github.com/user-attachments/assets/c11649f7-2920-44f3-be42-c37ce2cd8af4)

Reply:
@clairesunstudio thanks for the review. This is an edge case from my point of view, that is why I was so specific in my CSS selector. 
Agree that we need to test it on different screen sizes and probably try `style-position: inside`. (I like that solution more). 
If the image floats right - there is no issue with bullet points, because bullet points are on the left:
![Mass gov dev — (L) SSC Service Catalog Offerings TEST | Mass gov 2024-08-23 09-25-56](https://github.com/user-attachments/assets/83387571-f257-4979-82a3-0a5df8cfb358)
Also, it is not an issue if the image is full width:
![Mass gov dev — (L) SSC Service Catalog Offerings TEST | Mass gov 2024-08-23 09-28-43](https://github.com/user-attachments/assets/772fb1f2-77a2-4dea-b834-1ebd40371a98)

So I would stay with the selector `a__rich-text div.align-left.half-width.embedded-entity + ul` and try `style-position: inside`.

@joeg8612, what do you think this is an edge case and not a widely used pattern, or we want to make this available for all Mayflower dependent sites?